src/core_modules/capture-core/reducers/descriptions/searchPage.reducerDescription.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import { createReducerDescription } from '../../trackerRedux/trackerReducer';
+import { searchPageActionTypes } from '../../components/Pages/Search/SearchPage.container';
+
+export const searchPageStatus = {
```

Comment:
I am using the term "Status" here to signify the state of the page. 

In this case a status is like a machine state.
![image](https://user-images.githubusercontent.com/4181674/86750726-13223f80-c036-11ea-92c9-5b514782a112.png)

That means that the Search page would render specific content given the statuses.